rep-0146.rst

```diff
@@ -0,0 +1,61 @@
+REP: 146
+Title: ROS Robot Parameter Naming Convention
+Author: Kei Okada <k-okada@jsk.t.u-tokyo.ac.jp>
+Status: Draft
+Type: Informational
+Content-Type: text/x-rst
+Created: 31-Jly-2015
+Post-History:
+
+Abstract
+========
+
+This REP aims to define the best practices for robot related parameter
+name sapce for making robot-agnostic packages.
+
+Motivation
+==========
+
+As the number of ROS-supporeted robot increases, we will write
+robot-agnostics packages, but still they need robot-specific
+informations as calibration parametesrs.
+
+A `robot_description` parameter is widly shared naming convention
+among ROS cummnity and this REP proposes such conventions for robot specific information on
+parameter, so application program can build their program on top of that.
+
+
+Specifications
+==============
+
+Paramters
+---------
+
+robot_description
+'''''''''''''''''
+
+`robot_description` stores robot kinematics / volumentric information as
+text format, that can be either URDF or COLLADA file format
```

Comment:
ok, removed collada, but since robot_model library can read both urdf and collada, so i think most of core tools using that library can support that. (rviz, robot_state_publisher ...)
https://github.com/ros/robot_model/blob/indigo-devel/urdf/src/model.cpp#L142